com.unity.render-pipelines.high-definition/Editor/AssetProcessors/MaterialPostProcessor.cs

```diff
@@ -128,15 +309,62 @@ static void OnPostprocessAllAssets(string[] importedAssets, string[] deletedAsse
         {
             foreach (var asset in importedAssets)
             {
-                if (!asset.ToLowerInvariant().EndsWith(".mat"))
+                // We intercept shadergraphs just to add them to s_ImportedAssetThatNeedSaving to make them editable when we save assets
+                if (asset.ToLowerInvariant().EndsWith($".{ShaderGraphImporter.Extension}"))
```

Comment:
Related to https://github.com/Unity-Technologies/Graphics/pull/3988/files#r600523643